### PR TITLE
addEven

### PR DESCRIPTION
**Pull request by sxlivan**

### CODE REVIEW

File: homework/add-even/addEven.hpp [hunk 1]
```diff
@@ -2,7 +2,11 @@
 #include <vector>
 
 int addEven(const std::vector<int>& numbers) {
-    // TODO: Your implementation goes here
-    // Below return is only to make this function compile now
-    return -1;
+    int sum = 0;
+    for(auto i : numbers){
```
Comment (BarTes8) [inline homework/add-even/addEven.hpp:6]: could write for (const auto i : numbers) {

File: homework/add-even/addEven.hpp [hunk 2]
```diff
@@ -2,7 +2,11 @@
...
+        if(i%2==0){
```
Comment (BarTes8) [inline homework/add-even/addEven.hpp:7]: add spaces if (i % 2 == 0) {

**Comment by ziobron** [comment]
Thanks @BarTes8 for Code Review.
🏅 1 XP granted

**Comment by ziobron** [comment]
Thanks @BarTes8 for Code Review.
🏅 1 XP granted

**Comment by ziobron** [comment]
Thanks @BarTes8 for Code Review.
🏅 1 XP granted

**Pull request closed without merging**